lib/job/arGenerateReportJob.class.php

```diff
@@ -155,7 +155,19 @@ private function getStorageLocationsResults()
     $criteria->addJoin(QubitRelation::OBJECT_ID, QubitInformationObject::ID);
     $criteria->addJoin(QubitRelation::SUBJECT_ID, QubitPhysicalObject::ID);
 
-    return QubitPhysicalObject::get($criteria);
+    $storageLocations = QubitPhysicalObject::get($criteria);
+    $results = array();
+
+    foreach($storageLocations as $item)
```

Comment:
I think some logic in the template in this case would be better, to avoid looping twice over the results. Not user if this could be fixed in the render methods and getting the values with source culture fallback in the templates.